daemon/cmd/daemon.go

```diff
@@ -899,8 +899,8 @@ func NewDaemon(ctx context.Context, cancel context.CancelFunc, epMgr *endpointma
 		}
 
 		if option.Config.EnableL7Proxy {
-			return nil, nil, fmt.Errorf("egress gateway requires L7 proxy to be disabled (--%s=\"false\").",
-				option.EnableL7Proxy)
+			log.Warning("both egress gateway and L7 proxy are enabled. This is currently not fully supported: " +
```

Comment:
Maybe
```suggestion
			log.Warning("both %s and %s are enabled. This is currently not fully supported: " +
			"if the same endpoint is selected both by an egress gateway and a L7 policy, endpoint traffic will not go through egress gateway.", option.EnableIPv4EgressGateway, option.EnableL7Proxy)
```

Reply:
updated while trying to keep it consistent with the other messages :+1: